sysid/src/main/native/include/sysid/view/Analyzer.h

```diff
@@ -91,6 +93,11 @@ class Analyzer : public glass::View {
    */
   void AnalyzeData();
 
+  /**
+   * Used by DataSelector to import any missing tests.
+   */
+  void SetMissingTests(std::vector<std::string>);
```

Comment:
Include an argument name here. Also, you should probably pass by const ref if you aren't going to use std::move().